### DISCUSSION TITLE
Move highb/asdf-teleport-ent to community repo

### DISCUSSION THREAD

**Discussion by highb** [asdf-community/.github#53] [General]
I just wrote a plugin for managing Teleport enterprise CLI tool versions (not affiliated with the company, just a heavy user) and I'd like to move my plugin into https://github.com/asdf-community , if you'll have it. Happy to provide any additional details or context on it. Thanks!

**Comment by highb** [comment]
I'll open a PR, as well.

**Comment by highb** [comment]
PR opened: https://github.com/asdf-vm/asdf-plugins/pull/570

**Comment by smorimoto** [comment]
Consult this discussion post: https://github.com/orgs/asdf-community/discussions/57 This is the smoothest way to go. Of course I can do all the work for you, but it's done on a best effort basis. (I've been busy with work lately...)
